pkg/server/serverpb/admin.proto

```diff
@@ -399,6 +399,13 @@ message DrainRequest {
   // shutdown for an explanation of the interaction between the two.
   // do_drain is also implied by a non-nil deprecated_probe_indicator.
   bool do_drain = 4;
+
+  // node_id is a string so that "local" can be used to specify that no
+  // forwarding is necessary.
+  // For compatibility with v21.2 nodes, an empty node_id is
+  // interpreted as "local". This behavior might be removed
+  // in subsequent versions.
+  string node_id = 5;
```

Comment:
What's the argument against using our usual roachpb.NodeID type here, but overloading the zero value as a request to drain the node this request is issue to? It would also be backwards compatible with v21.2.